apps/smart-contracts/core/scripts/ArbitragePools.ts

```diff
@@ -283,3 +283,15 @@ export const getAdjustedEstimate = (
   }
   throw new Error('Cannot get estimate for INSUFFICIENT_SPREAD')
 }
+
+/**
+ * `growthFactor` is a string because BigNumber does not support decimal
+ * inputs for calculations. Instead, we convert a float string,
+ * e.g. '1.05'(1.05x), to a 18 decimal factor.
+ */
+export const getNextTradeSize = (currentTradeSize: BigNumber, growthFactor: string): BigNumber => {
+  if (parseEther(growthFactor).lt(0)) {
```

Comment:
```suggestion
  if (parseEther(growthFactor).lte(1)) {
```
if <1, then it will shrink. if =1 it won't grow

add tests for =0, =1, and right below 1